packages/protocol/contracts/identity/test/TestAttestations.sol

```diff
@@ -0,0 +1,25 @@
+pragma solidity ^0.5.3;
+
+import "../Attestations.sol";
+
+
+/*
+ * We need a test contract that behaves like the actual Attestations contract,
+ * but mocks the implementations of the validator set getters. Otherwise we
+ * couldn't test `request` with the current ganache local testnet.
+ */
+contract TestAttestations is Attestations {
+  address[] private __testValidators;
```

Comment:
Why `__` prefix? One underscore would be more consistent with what we do elsewhere

Reply:
That was precisely my reasoning - since we use `_` sometimes, since this is a mock function that should never collide with an actual function, `__` would avoid such potential collisions. Of course we don't want to fall down the rabbit hole of `___`, `____`, ..., in the future, but I can't imagine us needing to go down another level.

Reply:
Do we expect collisions? I feel like a single `_` is fine